maven-core/src/main/java/org/apache/maven/project/ReactorModelCache.java

```diff
@@ -113,7 +107,48 @@ public int hashCode()
         {
             return hashCode;
         }
+    }
+    
+    private static final class GACacheKey
+    {
+        private final String groupId;
+        
+        private final String artifactId;
+        
+        private final int hashCode;
 
+        GACacheKey( String groupId, String artifactId )
+        {
+            super();
```

Comment:
This super is is redundant, isn't it?

Reply:
yes, it is there already implicitly, but I'll remove it.